Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabPublicHandler.cpp

```diff
@@ -279,6 +292,8 @@ namespace AzToolsFramework
 
         CreatePrefabResult PrefabPublicHandler::CreatePrefabInDisk(const EntityIdList& entityIds, AZ::IO::PathView filePath)
         {
+            AZ_Assert(filePath.IsAbsolute(), "CreatePrefabInMemory requires an absolute file path.");
```

Comment:
Please change this to reflect the `CreatePrefabInDisk` name (this was copy/pasted from `CreatePrefabInMemory`? sort of funny we require a file path for that)